tendrl/node_agent/flows/execute_command.py

```diff
@@ -0,0 +1,13 @@
+from tendrl.node_agent.atoms.generic.command import Command
+
+
+class ExecuteCommand(object):
+    def __init__(self, api_job):
+        super(ExecuteCommand, self).__init__()
+        self.api_job = api_job
+        self.atom = Command
+
+    def start(self):
+        attributes = self.api_job['attributes']
+        result, err = self.atom().start(attributes)
```

Comment:
Shouldnt we take _raw_input from attributes and pass to start to be specific. I understand inside atom's start() function you might be doing that.